tests/test_language_en.py

```diff
@@ -0,0 +1,145 @@
+from number_parser import parser
+import pytest
+
+LANG = 'en'
+
+
+class TestNumberParser():
+    @pytest.mark.parametrize(
+        "expected,test_input",
+        [
+            (2003984, "two million three thousand nine hundred and eighty four"),
+            (19, "nineteen"),
+            (2019, "two thousand and nineteen"),
+            (2003019, "two million three thousand and nineteen"),
+            (3000000000, 'three billion'),
+            (3000000, 'three million'),
+            (123456789, 'one hundred twenty three million four hundred fifty six thousand seven hundred \
+                and eighty nine'),
+            (11, 'eleven'),
+            (19000000019, 'nineteen billion and nineteen'),
+            (142, 'one hundred and forty two'),
+            (2023049, 'two million twenty three thousand and forty nine'),
+            (100, 'hundred'),
+            (1000, 'thousand'),
+            (1000000, 'million'),
+            (1000000000, 'billion'),
+
+        ]
+    )
+    def test_parse_number(self, expected, test_input):
+        assert parser.parse_number(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "expected,test_input",
+        [
+            (100000, "100000"),
+            (None, "1 2"),
+            (None, "twenty 1"),
+            (12341252352314, "12341252352314"),
+            (21, 'twenty-one'),
+            (365, 'three-hundred and sixty-five'),
+        ]
+    )
+    def test_parse_number_digits(self, expected, test_input):
+        assert parser.parse_number(test_input, LANG) == expected
+
+    @pytest.mark.parametrize(
+        "test_input,expected",
+        [
+            ("twenty-five cows, twelve chickens and one hundred twenty five kg of potatoes.",
+                "25 cows, 12 chickens and 125 kg of potatoes."),
+
+            ("I have eight cows",
+                "I have 8 cows"),
+
+            ("I have eight cows three bulls and seven hundred and twelve million dollars ",
+                "I have 8 cows 3 bulls and 712000000 dollars"),
+
+            ("They just won seventy-five thousand dollars",
+                "They just won 75000 dollars"),
+
+            ("I have eight cows. I don't have eighteen cows",
+                "I have 8 cows. I don't have 18 cows"),
+
+            ("thirty-four cows = thirty four cows",
+                "34 cows = 34 cows"),
+
+            ("thirty       four       cows = thirty four cows",
+                "34 cows = 34 cows"),
+
+            ("the     sun      is     hundred     and   twelve       km     away.",
+                "the     sun      is     112 km     away."),
+        ]
+    )
+    def test_basic_sentences(self, expected, test_input):
```

Comment:
It's a good pattern to write the name of the tested function after the `test_`. I would rename this function to:

```suggestion
    def test_parse_basic_sentences(self, expected, test_input):
```

The same for the other tests: 
* `test_case_of_string` --> `test_parse_case_of_string`
* `test_ambiguity_in_multipliers` --> `test_parse_ambiguity_in_multipliers`
* `test_ambiguity_in_separators` --> `test_parse_ambiguity_in_separators`

and so on.